src/speechly/client.ts

```diff
@@ -75,7 +76,7 @@ export class Client {
   private intentCb: IntentCallback = () => {}
 
   constructor(options: ClientOptions) {
-    if (!localeCode.validate(options.language)) {
+    if (options.language != null && !localeCode.validate(options.language)) {
```

Comment:
Hm, this will work, but it's now possible for `defaultLanguage` to be invalid. While the risk is quite low, I still suggest that you validate the default one as well.

Something like this should work:
```suggestion
    const language = options.language ?? defaultLanguage
    if (!localeCode.validate(language)) {
```